### PR TITLE
add custom function sample in remote mode

### PR DESCRIPTION
**Pull request by krokhaleva**

### CODE REVIEW

**Comment by beebs-systap** [comment]
Computer, please test this.

File: sample-tests/src/test/resources/sampleweb.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<web-app xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://java.sun.com/xml/ns/javaee" xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_3_0.xsd" id="WebApp_ID" version="3.0">
+     
+  <listener>
+    <listener-class>sample.customFunction.remote.MyBigdataRDFContextListener</listener-class>
```
Comment (thompsonbry) [inline sample-tests/src/test/resources/sampleweb.xml]: @beebs-systap has recently fixed the way in which web-override.xml is found. This feature works correctly in the 2.0 RC.  It should be used for these examples.

File: sample-customFunction-remote/src/main/resources/jettyMavenTest.xml [hunk 2]
```diff
@@ -0,0 +1,171 @@
+<?xml version="1.0"?>
```
Comment (thompsonbry) [inline sample-customFunction-remote/src/main/resources/jettyMavenTest.xml]: @beebs-systap @igor-kim @krokhaleva Do we need this if we are using web-override.xml?

File: sample-tests/src/test/resources/sampleweb.xml (hunk 1 above)
Comment (beebs-systap) [inline sample-tests/src/test/resources/sampleweb.xml]: Yes, should be able to set the `SystemProperties.JETTY_OVERRIDE_WEB_XML` property in your main and achieve the same result.  See https://github.com/SYSTAP/bigdata/pull/265/

**Comment by beebs-systap** [comment]
Computer, please test this.

**Comment by beebs-systap** [comment]
CI is clean for the 1.5.3 release.   Please update and re-test with the 2.0.0-SNAPSHOT.  See https://github.com/blazegraph/blazegraph-samples/pull/11 which has been merged with this branch.

**Comment by beebs-systap** [comment]
Merged into 2.0.0 release.

**Pull request closed without merging**